dnfdaemon-server/dbus.hpp

```diff
@@ -39,7 +39,8 @@ using DbusTransactionItem = sdbus::Struct<
     std::string,  // version
     std::string,  // release
     std::string,  // arch
-    std::string   // repoid
+    std::string,  // repoid
+    unsigned int  // size
```

Comment:
Not necessarily in this PR - what do you think about unifying the ways how packages are transfered via dbus? I mean instead of `a(ussssssu)` use dictionary `a(ua{sv})`- like the `list()` method does. It is more flexible and allows us for example to return another package field in the future without changing the method signature.
If you are not changing it in this PR, please leave a `TODO` and file a jira issue to have it tracked.